### DISCUSSION TITLE
How to hard reload chrome without caching resources?

### DISCUSSION THREAD

**Discussion by rahul-raut** [webdriverio/webdriverio#7304] [Q&A]
Is there any way to hard reload page using webrdiverIO? 
I tried `browser.keys ('Control', 'Shift', 'F5')` doesn't work as expected.
I tried chrome-options by sending --disable-application-cache and --disk-cache-size. 
Please help

**Comment by christian-bromann** [comment]
use https://webdriver.io/docs/api/webdriver#refresh

**Reply by rahul-raut** [reply]
No it's not hard reset, it's using memory cache. I wanted hard reload without any cache.

**Reply by christian-bromann** [reply]
then `browser.execute(() => location.reload(true))`

**Reply by rahul-raut** [reply]
That also did not hard reload, I have tried that as well.

**Reply by rahul-raut** [reply]
Well I found another way where I can switch to chrome's clear cache url "chrome://settings/clearBrowserData" and wrote a script to click on clear cache button. That solved my problem. Thanks for the help!
